common-lib/src/main/scala/com/gu/mediaservice/lib/guardian/GuardianUsageRightsConfig.scala

```diff
@@ -103,25 +113,25 @@ object GuardianUsageRightsConfig extends UsageRightsConfigProvider {
 
   val contractIllustrators = List(
     PublicationPhotographers(GuardianPublication, List(
-      "Ben Lamb",
-      "Andrzej Krauze",
-      "David Squires",
-      "First Dog on the Moon",
-      "Harry Venning",
-      "Martin Rowson",
-      "Matt Kenyon",
-      "Matthew Blease",
-      "Nicola Jennings",
-      "Rosalind Asquith",
-      "Steven Appleby",
-      "Ben Jennings",
+      PublicationPhotographer("Ben Lamb"),
+      PublicationPhotographer("Andrzej Krauze"),
+      PublicationPhotographer("David Squires"),
+      PublicationPhotographer("First Dog on the Moon"),
+      PublicationPhotographer("Harry Venning"),
+      PublicationPhotographer("Martin Rowson"),
+      PublicationPhotographer("Matt Kenyon"),
+      PublicationPhotographer("Matthew Blease"),
+      PublicationPhotographer("Nicola Jennings"),
+      PublicationPhotographer("Rosalind Asquith"),
+      PublicationPhotographer("Steven Appleby"),
+      PublicationPhotographer("Ben Jennings"),
       //Past
-      "Steve Bell",
+      PublicationPhotographer("Steve Bell"),
     )),
     PublicationPhotographers(ObserverPublication, List(
-      "Chris Riddell",
-      "David Foldvari",
-      "David Simonds",
+      PublicationPhotographer("Chris Riddell"),
+      PublicationPhotographer("David Foldvari"),
+      PublicationPhotographer("David Simonds"),
```

Comment:
is the reason why these Obs people don't have `to` dates, whereas the people in `ContractedPhotographers` (mostly) do, because we don't expect any new images to come from these three, whereas the contracted photographers are liable to submit new Guardian images? Or am I misunderstanding something about the logic here?